examples/next/getting-started/config.js

```diff
@@ -0,0 +1,9 @@
+const { headlessConfig } = require('@faustjs/core');
```

Comment:
Nit: Maybe this be `faust.config.js` for clarity?

Reply:
I support this as well, I think this is initially what @blakewilson was going to call it.